content/developer/reference/backend/testing.rst

```diff
@@ -730,14 +731,113 @@ To start a tour from a python test, make the class inherit from
 
    def test_your_test(self):
        # Optional Setup
-       self.start_tour("/web", 'your_module.your_tour_name', login="admin")
+       self.start_tour("/web", "your_tour_name", login="admin")
        # Optional verifications
 
+Writing an onboarding tour
+--------------------------
+
+The main purpose of the onboarding tours is to guide the user. They can be started with the
+checkbox in the user menu that will run all the onboarding tours in their sequence order.
+
+.. image:: testing/tour_user_menu.png
+   :align: center
+
+Structure
+~~~~~~~~~
+
+To write an onboarding tour for `your_module`, start with creating the required files:
+
+.. code-block:: text
+
+    your_module
+    ├── ...
+    ├── data
+    |   └── your_tour.xml
+    ├── static
+    |   └── src
+    |       └── js
+    |           └── tours
+    |               └── your_tour.js
+    └── __manifest__.py
+
+You can then update :file:`__manifest__.py` to add :file:`your_tour.js` in the assets and :file:`your_tour.xml` in the data.
+
+  .. code-block:: python
+
+     'data': [
+        'data/your_tour.xml',
+     ],
+     'assets': {
+         'web.assets_backend': [
+             'your_module/static/src/js/tours/your_tour.js',
+         ],
+     },
+
+Javascript
+~~~~~~~~~~
+
+The javascript part is the same as for the test tour.
+
+.. seealso::
+   - :ref:`testing/javascript/test`
+
+XML
+~~~
+
+When you have your tour in the javascript registry, you can create a record `web_tour.tour` in the xml, like that:
+
+  .. code-block:: xml
+
+      <?xml version="1.0" encoding="utf-8"?>
+      <odoo>
+          <record id="your_tour" model="web_tour.tour">
+              <field name="name">your_tour</field>
+              <field name="sequence">10</field>
+              <field name="rainbow_man_message">Congrats, that was a great tour</field>
+          </record>
+      </odoo>
+
+- **name**: Required, the name must be the same as the one in the
+  javascript registry.
+- **sequence**: Optional, will determine the order to execute the
+  onboarding tours, If no ``sequence``, has a default value of 1000.
+- **url**: Optional, the url where to start the tour. If no ``url``,
+  take the url from the registry.
+- **rainbow_man_message**: Optional, will show the message in the
+  rainbow man effect at the completion of the tour. If no ``rainbow_man_message``,
+  there is a default value. If you do not want a rainbow_man effect, then specify
+  the field and write nothing as value.
+
+Running onboarding tour
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Like said previously, you can activate the onboarding tour with the checkbox in user menu,
+but you can also run specific ones through the menu: ``Settings/Technical/User Interface/Tours``.
+In this view you'll get all the onboarding tours and you can execute them by clicking on `onboarding` or `testing`.
```

Comment:
With moving the intro here in mind:
```suggestion
You can run specific onboarding tours by going to the :menuselection:`Settings --> Technical --> User Interface --> Tours` and clicking on :guilabel:`Onboarding` or :guilabel:`Testing`.
```

Reply:
Given that this explanation is straightforward, I think that the second image should be removed too.